metagpt/utils/repair_llm_raw_output.py

```diff
@@ -105,6 +105,23 @@ def judge_potential_json(routput: str, left_key: str) -> Union[str, None]:
     return output
 
 
+def remove_comments_from_line(line):
```

Comment:
add unittest case for `//` like `https://github.com/geekan/MetaGPT/blob/main/tests/metagpt/utils/test_repair_llm_raw_output.py#L131-L141`

Reply:
OK

Reply:
Modify code based on feedback of action_node.py and repair_llm_raw_output.py, add code in test_repair_llm_raw_output.py. Please have a look, thanks.